src/config.ts

```diff
@@ -91,11 +92,15 @@ const mustHave = {
   IMPULSE_APPLICATION_ID,
   POSITRON_API_BASE,
   EXCHANGE_API_BASE,
+  KAWS_API_BASE,
```

Comment:
This puts it in `mustHave` which means tests/app won't load unless you've set it, and will give a useful error message